src/components/common/notification-center.svelte

```diff
@@ -0,0 +1,51 @@
+<script>
+  import { onMount } from 'svelte';
+  import Dropdown from 'ui/dropdown.svelte';
+  import Button from 'ui/button.svelte';
+  import * as api from '@/utils/api.js';
+
+  export let notifications = null;
+  $: unread = notifications != null && notifications.some(notification => !notification.is_read);
```

Comment:
`unread` will be false if `notifications == null`. If there are no notifications, then there are no unread notifications
Using `||` will also short-circuit if they are null
```suggestion
  $: unread = notifications == null || notifications.some(notification => !notification.is_read);
```

Reply:
If notifications are null, then the first condition will fail and `false` will be returned without evaluating the second condition, therefore, there will not be unread notifications. Feels like the same thing

Reply:
Oh. seems like I had it the other way around. nevermind.